src/ascii.rs

```diff
@@ -23,7 +155,7 @@ impl Ascii {
     #[inline]
     pub fn from(ch: char) -> Result<Ascii, ()> {
         if ch as u32 <= 0x7F {
-            return Ok( Ascii { chr: ch as u8 });
+            return Ok(unsafe{ ch.to_ascii_nocheck() });
```

Comment:
Personally I would wrap the complete if-statement in the unsafe block. The if-clause is vital to the correctness of `ch.to_ascii_nocheck()`.

Do you mind changing it?